assets/embedded-files/epinio/basicauth.yaml

```diff
@@ -7,3 +7,4 @@ metadata:
 spec:
   basicAuth:
     secret: epinio-api-auth-secret
+    headerField: X-WebAuth-User
```

Comment:
It seems that if we ever want to support another middleware (e.g. forwardauth) we would need to check for some other header instead: https://doc.traefik.io/traefik/v2.0/middlewares/forwardauth/#authresponseheaders

Anyway, let's think about it then.

Reply:
Agreed.